src/sdk-reference/js/6/kuzzle/introduction/index.md

```diff
@@ -50,11 +50,50 @@ Available properties.
 | `replayInterval`     | <pre>number</pre>  | Delay between each replayed requests               |    Yes    |
 | `volatile`           | <pre>object</pre> | Common volatile data, will be sent to all future requests       |    Yes    |
 
-**Notes:**
-
-- multiple methods allow passing specific `volatile` data. These `volatile` data will be merged with the global Kuzzle `volatile` object when sending the request, with the request specific `volatile` taking priority over the global ones.
-- the `queueFilter` property is a function taking an object as an argument. This object is the request sent to Kuzzle, following the [Kuzzle API]({{ site_base_path }}api/1/essentials/query-syntax) format
-- if `queueTTL` is set to `0`, requests are kept indefinitely
-- The offline buffer acts like a first-in first-out (FIFO) queue, meaning that if the `queueMaxSize` limit is reached, older requests are discarded to make room for new requests
-- if `queueMaxSize` is set to `0`, an unlimited number of requests is kept until the buffer is flushed
-- the `offlineQueueLoader` must be set with a function, taking no argument, and returning an array of objects containing a `query` member with a Kuzzle query to be replayed, and an optional `cb` member with the corresponding callback to invoke with the query result
+### offlineQueueLoader
+
+The `offlineQueueLoader` property must be set with a function of one of the following formats:
+
+```js
+Array<object> offlineQueueLoader()
```

Comment:
Shall we use the `object[]` notation instead ?

Reply:
Since this is a callback signature documentation, it seems to me that a jsdoc notation is preferable. But I don't have a strong opinion on that one.

Reply:
IMHO it feels weird to have two different notation for array, I think we should keep the same notation everywhere

Reply:
Fine by me. I'm updating these signatures.